spec/latest/core/index.html

```diff
@@ -1017,6 +1017,77 @@ <h3>Templated Links</h3>
       }
       -->
     </pre>
+
+    <p>Due to fact that hydra is on top of an RDF, which is a graph,
+      it may happen that a related resource (an object of the relation)
+      may not be fully described in the resource's payload. In case of
+      an <i>IriTemplate</i> expected as a related resource, client may
+      discover no additional statements describing it. These rules should
+      be considered when working with <i>IriTemplate</i>s (and other
+      hypermedia resources in general):
+    </p>
+
+    <ul>
+      <li>in case of an object expected to be a hypermedia resource does not have all
+        the necessary statements for which it is a subject, client SHOULD look in the
+        API documentation for more details</li>
+      <li>in case the mentioned object after consulting an API documentation still
+        does not have all the necessary statements for which it is a subject and both
+        mentioned object's Url and Url of the initially obtained resource has the
+        same scheme and authority (by means of RFC 3986 sections 3.1 and 3.2), client
+        SHOULD dereference that Url</li>
```

Comment:
```suggestion
        same scheme and authority (by means of RFC 3986 sections 3.1 and 3.2), client
        SHOULD dereference that Url. If the resource does not have the same scheme and authority,
        the client MAY choose to dereference it (for example if the resource originates from another
        API well-known to the client)</li>
```

I propose to add a `MAY` option for the client to dereference resources from other APIs if they so choose